hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/rule/ExcludeSpanRuleInfo.java

```diff
@@ -16,21 +17,27 @@ public interface ExcludeSpanRuleInfo {
 
   @GraphQLField
   @GraphQLName(NAME_KEY)
+  @GraphQLDescription("Exclude span rule name")
   @GraphQLNonNull
   String name();
 
   @GraphQLField
   @GraphQLName(SPAN_FILTER_KEY)
+  @GraphQLDescription("Span processing rule filter contains filters based on span attributes")
   @GraphQLNonNull
   SpanProcessingRuleFilter spanFilter();
 
   @GraphQLField
   @GraphQLName(DISABLED_KEY)
+  @GraphQLDescription(
+      "Disabled field denotes whether the rule is disabled or not. By default it is enabled on write")
```

Comment:
```suggestion
      "Disabled field denotes whether the rule is disabled or not.")
```

I meant this is the returned object on read - the output, not the input. The ExcludeSpanRuleUpdate (and Create - which seems to be missing in this PR) type is where you should describe defaults.

That said, now that I actually look at these types - the update API marks this as non-nullable. That means there isn't a default in the first place, it has to be explicitly assigned by the caller.